numpy/core/src/multiarray/ctors.c

```diff
@@ -1939,6 +1953,14 @@ PyArray_FromArray(PyArrayObject *arr, PyArray_Descr *newtype, int flags)
            !PyArray_EquivTypes(oldtype, newtype);
 
     if (copy) {
+
+        if( flags & NPY_ARRAY_ENSURENOCOPY ) {
+            PyErr_SetString(PyExc_ValueError,
+                            "Unable to avoid copy while creating "
+                            "an array from given array.");
```

Comment:
```suggestion
                    "Unable to avoid copy while creating an array from given array.");
```
Super nitpicky though, but we use that style a lot, and I still like it ;).